environment/src/test/java/com/sequenceiq/environment/environment/v1/converter/EnvironmentApiConverterTest.java

```diff
@@ -292,6 +293,35 @@ void testInitLoadBalancerDto() {
         assertEquals(subnetIds, environmentLbDto.getEndpointGatewaySubnetIds());
     }
 
+    @Test
+    void testAwsDiskEncryptionParametersAndAwsRequest() {
+        EnvironmentRequest request = createEnvironmentRequest(AWS);
+        request.setAws(AwsEnvironmentParameters.builder()
+                .withAwsDiskEncryptionParameters(
+                        AwsDiskEncryptionParameters.builder()
+                                .withEncryptionKeyArn("dummy-key-arn")
+                                .build())
+                .build());
```

Comment:
`EncryptionKeyArn` gets set as part of `createEnvironmentRequest`. Need not to set it again.